src/StreamFactoryDiscovery.php

```diff
@@ -3,7 +3,6 @@
 namespace Http\Discovery;
 
 use Http\Discovery\Exception\DiscoveryFailedException;
-use Http\Discovery\Exception\NotFoundException;
```

Comment:
hm. should we have the phpdoc say it throws the new exception? people should start relying on the new exception, not the deprecated one.